app/presenters/specialist_document_presenter.rb

```diff
@@ -22,6 +22,10 @@ def important_metadata
     end
   end
 
+  def show_metadata_block?
+    content_item.to_h.dig("links", "finder", 0, "details", "show_metadata_block")
```

Comment:
I think this code should be in the `Finder` model.

So create an attribute in the that model called `show_metadata_block` that does this `dig`

Then this method could be changed to call

`finder.show_metadata_block`

Reply:
We're keeping model and presenter code separate where possible, so anything that does a `dig` on `content_store_response` should be in the model.

Reply:
I've made the suggested changes 🙂